<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KK牌计算器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_QT.qrc">
    <normaloff>:/images/calculator.ico</normaloff>:/images/calculator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_three">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_paren2">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_six">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_eight">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_mult">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_point">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_four">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_sub">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_zero">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_seven">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_add">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_del">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_divide">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_equal">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_paren1">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_two">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_nine">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_one">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_five">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_C">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_std">
    <property name="title">
     <string>标准</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_sci">
    <property name="title">
     <string>科学</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_prog">
    <property name="title">
     <string>程序员</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_data">
    <property name="title">
     <string>日期计算</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_rate">
    <property name="title">
     <string>汇率计算</string>
    </property>
   </widget>
   <addaction name="menu_std"/>
   <addaction name="menu_sci"/>
   <addaction name="menu_prog"/>
   <addaction name="menu_data"/>
   <addaction name="menu_rate"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_C</tabstop>
  <tabstop>pushButton_paren1</tabstop>
  <tabstop>pushButton_paren2</tabstop>
  <tabstop>pushButton_del</tabstop>
  <tabstop>pushButton_seven</tabstop>
  <tabstop>pushButton_eight</tabstop>
  <tabstop>pushButton_nine</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_four</tabstop>
  <tabstop>pushButton_five</tabstop>
  <tabstop>pushButton_six</tabstop>
  <tabstop>pushButton_sub</tabstop>
  <tabstop>pushButton_one</tabstop>
  <tabstop>pushButton_two</tabstop>
  <tabstop>pushButton_three</tabstop>
  <tabstop>pushButton_mult</tabstop>
  <tabstop>pushButton_zero</tabstop>
  <tabstop>pushButton_point</tabstop>
  <tabstop>pushButton_equal</tabstop>
  <tabstop>pushButton_divide</tabstop>
 </tabstops>
 <resources>
  <include location="res_QT.qrc"/>
 </resources>
 <connections/>
</ui>
